src/network/core/tcp_http.cpp

```diff
@@ -228,7 +228,7 @@ int NetworkHTTPSocketHandler::Receive()
 	for (;;) {
 		ssize_t res = recv(this->sock, (char *)this->recv_buffer + this->recv_pos, lengthof(this->recv_buffer) - this->recv_pos, 0);
 		if (res == -1) {
-			int err = GET_LAST_ERROR();
+			int err = NetworkGetLastError();
 			if (err != EWOULDBLOCK) {
 				/* Something went wrong... (104 is connection reset by peer) */
 				if (err != 104) DEBUG(net, 0, "recv failed with error %d", err);
```

Comment:
Would you mind, while at it, also making sure these use `NetworkGetLastError` ? Those numbers always mean so little to me ..